hack/feature-deploy.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
 
+export KUSTOMIZE_DIR="${KUSTOMIZE_DIR:-/tmp}"
+export KUSTOMIZE_BIN=$KUSTOMIZE_DIR/kustomize
+
+# Function: download the latest Kustomize binary to /tmp
+function get_kustomize_binary (){
+    pushd /tmp
+    curl -s "https://raw.githubusercontent.com/\
+kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+    popd
```

Comment:
i'd say opening Parentheses and cd'ing is more error resilient:
```
(cd /tmp; curl ...)
```

in the former case, if the curl command fail (and set -e is enabled)
you'll be left in that directory instead of returning